migrations/00278_contributions.sql

```diff
@@ -0,0 +1,622 @@
+-- +goose Up
+/***************************************************************/
+/*** SCRIPT AUTHOR: Andreas Gangsø (andreasgangso@gmail.com) ***/
+/***    CREATED ON: 2024-04-24T09:48:59.627Z                 ***/
+/***************************************************************/
+
+--- BEGIN CREATE SEQUENCE "public"."mediaitems_contributions_id_seq" ---
+
+
+CREATE SEQUENCE IF NOT EXISTS "public"."mediaitems_contributions_id_seq" 
+	INCREMENT BY 1 
+	MINVALUE 1
+	MAXVALUE 2147483647
+	START WITH 1
+	CACHE 1
+	NO CYCLE;
+
+ALTER SEQUENCE "public"."mediaitems_contributions_id_seq" OWNER TO bccm;
+GRANT SELECT ON SEQUENCE "public"."mediaitems_contributions_id_seq" TO bccm; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+GRANT USAGE ON SEQUENCE "public"."mediaitems_contributions_id_seq" TO bccm; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+GRANT UPDATE ON SEQUENCE "public"."mediaitems_contributions_id_seq" TO bccm; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
```

Comment:
This will fail as there is not `bccm` user. Should be `manager` IIRC. Check some of the other files otherwise